src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/CSharp/ImplementInterfaceAbstractClassCodeActionProvider.cs

```diff
@@ -72,7 +72,7 @@ internal class ImplementInterfaceAbstractClassCodeActionProvider : CSharpCodeAct
 
             if (diagnostics is null)
             {
-                return null;
+                return EmptyResult;
```

Comment:
@TanayParikh let me know if this was intentionally returning null. If so, I'll revert.

Reply:
LGTM. _May_ require updating the test.